cmd/crossbuild.go

```diff
@@ -180,9 +172,17 @@ type platformGroup struct {
 func (pg platformGroup) Build(repoPath string) error {
 	if platformsParam := strings.Join(pg.Platforms[:], " "); platformsParam != "" {
 		fmt.Printf("> running the %s builder docker image\n", pg.Name)
-		if err := docker("run --rm -t -v $PWD:/app", pg.DockerImage, "-i", repoPath, "-p", sh.Quote(platformsParam)); err != nil {
-			return err
+
+		cwd, err := os.Getwd()
+		if err != nil {
+			return errors.Wrapf(err, "Couldn't get current working directory")
```

Comment:
In old-school error handling, messages were supposed to start lowercase. Has this changed with Wrapf?